sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/SqlOperators.java

```diff
@@ -132,6 +132,8 @@
   public static final SqlOperator DATE_OP =
       createUdfOperator("DATE", BeamBuiltinMethods.DATE_METHOD);
 
+  // TODO: Fix Later
+  @SuppressWarnings("nullness")
   public static final SqlUserDefinedFunction CAST_OP =
       new SqlUserDefinedFunction(
           new SqlIdentifier("CAST", SqlParserPos.ZERO),
```

Comment:
What default value should be put here instead of null ?

Reply:
null is correct.